src/flamegraph/mod.rs

```diff
@@ -121,7 +123,10 @@ where
         };
 
         let samples = frame.end_time - frame.start_time;
-        let samples_txt = samples.thousands_sep();
+
+        // add thousands separators to `samples`
+        let _ = samples_txt_buffer.write_formatted(&samples, &Locale::en);
```

Comment:
Should we `.expect()` here since the writer should never error?

Reply:
`write_formatted` doesn't return a `Result`; it returns the number of bytes written. The code is written like that because I usually turn on a lint that prevents me from not using the return value of functions. If you think that's ugly style, though; just let me know and we can change the line to just plain old `samples_txt_buffer.write_formatted(&samples, &Locale::en);`.

Reply:
Nah, that's fine then!